pkgs/development/libraries/wxsqliteplus/default.nix

```diff
@@ -11,6 +11,10 @@ stdenv.mkDerivation rec {
     sha256 = "0mgfq813pli56mar7pdxlhwjf5k10j196rs3jd0nc8b6dkzkzlnf";
   };
 
+  postPatch = ''
+    sed -i '/WX_CLEAR_ARRAY/s/$/;/' src/{createtable,sqlite3table}.cpp
```

Comment:
This deletes the matching line, right?

Reply:
Nope, it finds every occurrence of `WX_CLEAR_ARRAY`, and adds a `;` at the end of each of those lines. The reason is that `WX_CLEAR_ARRAY` is an inline function instead of a macro in wxGTK 3.1.2+.

Reply:
It should be upstreamed.